LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 HQS Quantum Simulations GmbH
+   Copyright 2025 HQS Quantum Simulations GmbH
```

Comment:
We should check in with Basti regarding this update

Reply:
isn't it what we've always done so far?